Khan/react-redux
# [🔥AUDIT🔥] Remove 'prepare' script from package.json

🖍 _This is an audit!_ 🖍

## Summary:
webapp's static-service-PR workflow can get stuck trying to install react-redux because the build step fails on CI for some reason.  Since we've already built the files and checked them in, we can skip that step.

Issue: None

## Test plan:
- update webapp to use the new commit
- verify that the 'yarn install' step in static-service-PR is working more reliably
